sdcm/utils/common.py

```diff
@@ -2247,3 +2248,19 @@ def walk_thru_data(data, path: str) -> Any:
             continue
         current = current.get(name, None)
     return current
+
+
+def update_authenticator(nodes, authenticator='AllowAllAuthenticator', restart=True):
+    """
+    Update the authenticator of nodes, restart the nodes to make the change effective
+    """
+    for node in nodes:
+        node.remoter.sudo(
+            f"sed -ie 's/^authenticator:.*/authenticator: {authenticator}/g' /etc/scylla/scylla.yaml")
```

Comment:
this PR is against master, so please, remove the `sed` command.
for branch-2020.1 you will need to use exactly like is here:
```
node.remoter.sudo(
            f"sed -ie 's/^authenticator:.*/authenticator: {authenticator}/g' /etc/scylla/scylla.yaml")
```

because branch-2020.1 doesn't have the feature to open a remote file.
but this PR should have it

Reply:
Fixed, thanks